h/services/rename_user.py

```diff
@@ -57,6 +58,11 @@ def _purge_auth_tickets(self, user):
             .filter(models.AuthTicket.user_id == user.id) \
             .delete()
 
+    def _purge_tokens(self, user):
+        self.session.query(models.Token) \
+            .filter(models.Token.userid == user.userid) \
```

Comment:
We should make sure not to delete developer tokens though, these might be a bit of a pain to regenerate and change in all the places where a user might be using them. Which unfortunately means that we need to change the `userid` on the Token table for them...